### PR TITLE
add help overlay

### LINKED ISSUE #252: Add Help link (F1)
Add a Help menu item that links to an on-line resource (to be opened in Chrome).

Parallax will maintain the linked page to provide the help resources needed for users.  Initially, this will be a PBASIC Language Reference.

For this release, the link should be:  [http://www.parallax.com/go/PBASICHelp](http://www.parallax.com/go/PBASICHelp)

### PR DESCRIPTION
**Pull request by monteslu**
#### What's this PR do?

adds a help overlay
#### Where should the reviewer start?

the new view and action
#### How should this be manually tested?

pull down branch, build, install, and press F1 after launch
#### What are the relevant tickets?

fixes #252

### CODE REVIEW

File: src/views/help-overlay.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </div>
+        <OverlayFooter>
+          <Button onClick={hideOverlay}>OK</Button>
+        </OverlayFooter>
+      </Overlay>
+    );
+  }
+}
+
+module.exports = createContainer(HelpOverlay, {
```
Comment (phated) [inline src/views/help-overlay.js]: doesn't use the workspace store at all, can remove all of the `createContainer` stuff

File: src/views/help-overlay.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const helpStyle = {
+  position: 'relative'
+};
+
+const helpLink = 'http://www.parallax.com/go/PBASICHelp';
+
+class HelpOverlay extends React.Component {
+
+  constructor(...args){
```
Comment (phated) [inline src/views/help-overlay.js]: no need for the constructor if you aren't doing anything inside it

File: src/views/help-overlay.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const React = require('react');
+const { createContainer } = require('sovereign');
```
Comment (phated) [inline src/views/help-overlay.js]: remove

File: src/plugins/keyboard-shortcuts.js [hunk 4]
```diff
@@ -36,6 +37,7 @@ function keyboardShortcuts(app, opts, done){
   } = keypress;
 
   const {
+    showHelpOverlay,
```
Comment (phated) [inline src/plugins/keyboard-shortcuts.js]: was trying to keep all the `show*Overlay` methods next to each other for easier refactoring

**Comment by monteslu** [comment]
updated

**Pull request merged**